Cargo.toml

```diff
@@ -32,7 +32,5 @@ thread_local = { version = "1.1.0", optional = true }
 [dev-dependencies]
 version-sync = "0.9.2"
 
-[[example]]
-name = "ui"
-path = "examples/ui.rs"
-required-features = ["bevy/x11", "bevy/png", "bevy/bevy_wgpu"]
+[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
+bevy = { version = "0.5", default-features = false, features = ["bevy_wgpu", "x11", "png"] }
```

Comment:
Oh, this is great! I guess it also lets us tidy this feature matrix a bit
https://github.com/mvlabat/bevy_egui/blob/main/.github/workflows/check.yml#L34